### DISCUSSION TITLE
Alert threshold command

### DISCUSSION THREAD

**Discussion by Morsghost** [liquidctl/liquidctl#476] [Q&A]
I'm on an old Kraken X60 and I've configured liquidctl to replace CAM after years of trouble with it, so far the fan controls are working well so thanks for that. The only issue I'm having is that the default alert threshold of 45C is much too low, and the --alert-threshold command doesn't seem to be recognized. Does this work on legacy devices, is there any usage information I might be missing? Thanks.

**Comment by jonasmalacofilho** [comment]
Almost nothing seems to be known about the Kraken X40 & X60... in fact, until now I wasn't even sure our driver would really work with them, as even the most basic information available about those devices can be conflicting.

So, yeah, it's quite possible that the alert feature is not yet supported by that model, or that it might work differently or only support limited configuration.

Is that feature exposed through CAM?  If so, and if you can capture some traffic between it and the device with different alert thresholds and colors, I can try to spot what might need to be fixed in our driver.

**Reply by jonasmalacofilho** [reply]
Opened bug #477 for this... let's move the discussion there.
